apps/arweave/src/ar_http_iface_middleware.erl

```diff
@@ -861,7 +863,11 @@ verify_mempool_txs_size(MempoolTXs, TX) ->
 			valid
 	end.
 
-handle_post_tx_accepted(TX) ->
+handle_post_tx_accepted(PeerIP, TX) ->
+	%% Exclude successful requests with valid transactions from the
+	%% IP-based throttling, to avoid connectivity issues at the times
+	%% of excessive transaction volumes.
+	ar_blacklist_middleware:increment_ip_addr(PeerIP, -1),
```

Comment:
Prefer having a `ar_blacklist_middleware:decrement_ip_addr` method instead that does this.